### PR TITLE
Principal Menu

### PR DESCRIPTION
**Pull request by ismailbohra**

### CODE REVIEW

File: app/src/main/res/drawable/ic_baseline_home_repair_service_24.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
+    android:tint="?attr/colorControlNormal">
+  <path
```
Comment (MohitGupta121) [inline app/src/main/res/drawable/ic_baseline_home_repair_service_24.xml]: Please name properly of drawable

**Comment by MohitGupta121** [comment]
Also attach screenshot of your task

**Comment by ismailbohra** [comment]
[
principal_ui_1
principal_ui_2
](url)

**Comment by MohitGupta121** [comment]
@ismailbohra ok Done Great work

**Pull request merged**